Resources/Prototypes/Entities/Effects/Markers/trash_spawner.yml

```diff
@@ -17,17 +17,16 @@
     - type: TrashSpawner
       rarePrototypes:
         - PuddleVomit
-        - TrashPizzaBoxMessy
+        - PuddlePiss
+        - PuddleShit
```

Comment:
undocumented change

do we really want this?

Reply:
(The correct answer is no.)

Reply:
the correct answer is, indeed, *no*